udp/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package udp provides a UDP connection and server.
```

Comment:
```suggestion
// Package udp provides a UDP server implementation.
```

The conn wrapper is maybe just a detail IMO, the main point of the package is the provide a server implementation I guess? Same for TCP

Reply:
Well it looks like it's just the server, but previously the (TCP) connection was just implemented in the root package's client, which made it look like it's not needed, but it is as important as the server implementation to make it replaceable.